apps/vaporgui/VaporWidgets.h

```diff
@@ -138,6 +139,47 @@ private slots:
     std::string _text;
 };
 
+//
+// ====================================
+// VSlider augments QSlider by providing a text box displaying the value.
+//
+// Note: this widget is re-implemented by Sam instead of using the existing
+//   QSliderEdit class because:
+//   1) this class does NOT use a UI file following the VWidget convension;
```

Comment:
Yes, this class does not use a UI file.  However the VWidget convention does not use UI files either.